domain/serviceconfigfile/store.go

```diff
@@ -61,25 +72,36 @@ func (s *storeImpl) GetConfigFiles(ctx datastore.Context, tenantID string, svcPa
 		}
 
 		conv, err := convert(results)
+		confs = append(confs, conv...)
 		if err != nil {
 			return nil, err
-		} else if len(conv) == 0 {
+		} else if len(conv) == 0 || len(conv) < size {
```

Comment:
Why return here? Not sure this works like the original.

Reply:
This is a simple pulling by a batch mechanism. We're getting to return if the results no longer greater then the batch size. I hardly can't understand what else was conceived in the original.